README.md

```diff
@@ -2,7 +2,8 @@
 
 [![GitHub Release][github-release-img]][release]
 [![PSGallery Release][psgallery-release-img]][psgallery]
-[![GitHub CICD Workflow][github-cicd-workflow-img]][github-cicd-workflow]
+![CI Pipeline](https://github.com/cisagov/ScubaGear/actions/workflows/run_pipeline.yaml/badge.svg)
```

Comment:
Change these to point of reference links for consistency, examples are defined at the bottom of the current README.md or you can look at #494 as well.

![Screenshot (210)](https://github.com/user-attachments/assets/fec9a5d1-2a70-438f-bad8-68af01f2c82b)

Reply:
Done